src/forgot-password/tests/ForgotPasswordPage.test.jsx

```diff
@@ -72,31 +73,42 @@ describe('ForgotPasswordPage', () => {
       status: null,
     };
   });
+  const findByTextContent = (container, text) => Array.from(container.querySelectorAll('*')).find(
+    element => element.textContent === text,
+  );
 
   it('not should display need other help signing in button', () => {
-    const wrapper = mount(reduxWrapper(<IntlForgotPasswordPage {...props} />));
-    expect(wrapper.find('#forgot-password').exists()).toBeFalsy();
+    const { queryByTestId } = render(reduxWrapper(<IntlForgotPasswordPage {...props} />));
+    const forgotPasswordButton = queryByTestId('forgot-password');
+    expect(forgotPasswordButton).toBeNull();
   });
 
   it('should display need other help signing in button', () => {
     mergeConfig({
       LOGIN_ISSUE_SUPPORT_LINK: '/support',
     });
-    const wrapper = mount(reduxWrapper(<IntlForgotPasswordPage {...props} />));
-    expect(wrapper.find('#forgot-password').first().text()).toEqual('Need help signing in?');
+    render(reduxWrapper(<IntlForgotPasswordPage {...props} />));
+    const forgotPasswordButton = screen.findByText('Need help signing in?');
+    expect(forgotPasswordButton).toBeDefined();
   });
 
   it('should display email validation error message', async () => {
     const validationMessage = 'We were unable to contact you.Enter a valid email address below.';
-    const wrapper = mount(reduxWrapper(<IntlForgotPasswordPage {...props} />));
+    render(reduxWrapper(<IntlForgotPasswordPage {...props} />));
 
-    wrapper.find('input#email').simulate(
-      'change', { target: { value: 'invalid-email', name: 'email' } },
-    );
-    await act(async () => { await wrapper.find('button.btn-brand').simulate('click'); });
-    wrapper.update();
+    const emailInput = screen.getByLabelText('Email');
+
+    fireEvent.change(emailInput, { target: { value: 'invalid-email' } });
 
-    expect(wrapper.find('.alert-danger').text()).toEqual(validationMessage);
+    const submitButton = screen.getByText('Submit');
+    fireEvent.click(submitButton);
+
+    await waitFor(() => {
+      const alertElements = screen.getAllByRole('alert');
+      const validationErrors = alertElements.map(alertElement => alertElement.textContent)[0];
```

Comment:
can we do this? if we can't find directly by class?
```suggestion
      const validationErrors = alertElements[0].textContent;
```